Source/Mixers/iOS/GLESVideoMixer.swift

```diff
@@ -229,26 +82,25 @@ open class GLESVideoMixer: IVideoMixer {
         frameH = frame_h
         self.pixelBufferPool = pixelBufferPool
 
-        zRange.0 = Int.max
-        zRange.1 = Int.min
+        zRange.0 = .max
+        zRange.1 = .min
         
-        callbackSession = .init()
+        callbackSession = GLESObjCCallback()
         callbackSession.mixer = self
 
-        perfGLSync({
-            
+        perfGLSync(glContext: glesCtx, jobQueue: glJobQueue) {
             self.setupGLES(excludeContext: excludeContext)
```

Comment:
@thmatuza 
Is it necessary to weakify the reference of self here?

Reply:
Thank you for refactoring and pointing out it!
I believe it is not necessary to weakify the reference of self here because JobQueue class doesn't have reference to callback function.
However, GLESVideMixer::deinit is not called because of other bugs.
I will fix it.

Reply:
I fixed it the memory leak.
https://github.com/openfresh/VideoCast-Swift/pull/3